.github/workflows/main.yml

```diff
@@ -3,11 +3,11 @@ on:
   push:
     branches:
       - main
+  pull_request:
```

Comment:
Should we restrict this for PRs by specific users or PRs from branches in this repo (rather than on forks)?